etc/raintank/metric_tank-sample.ini

```diff
@@ -5,6 +5,6 @@ nsqd-tcp-address = localhost:4150,1.2.3.4:4150,5.6.7.8:4150
 cassandra-addrs = localhost
 listen = :6060
 # with these aggregation rules:
-# 5 min of data, store in a chunk that lasts 1hour, keep 2 chunks in memory
-# 1hr worth of data, in chunks of 6 hours, 2 chunks in mem
-agg-settings = 300:3600:2,3600:21600:2
+# 5 min of data, store in a chunk that lasts 1hour, keep 2 chunks in memory, purge after 12h
```

Comment:
instead of "purge after 12h" can we describe the ttl as "store in Cassandra for 12h" or similar.